README.md

```diff
@@ -23,396 +23,345 @@ Stable release versions are manually released from the main branch.
 
 ### Canary version
 
-A canary version is a pre-release version. Make sure all changes are added and committed before running the command below:
+A canary version is a pre-release version from `dev` branch.
+Make sure all changes are added and committed before running the command below:
 
 ```console
 npm run version:canary
 ```
 
 ## Usage
 
-### Define Staking Parameters
-
+> **Note**: This documentation describes an abstraction layer designed to simplify two main processes:
+> 1. Creating Bitcoin staking delegations
+> 2. Constructing data in the format required by the Babylon chain
+>
+> While this abstraction layer makes the process more straightforward, it comes with certain constraints:
```

Comment:
Why do we list the below here? The reader does not have any context when they reach this point. Better to list it when we describe the abstraction methods. For example, "This method X makes the change address be the same as the staker address. If you do not want to do that, you have to use lower level method Y".